### ISSUE TITLE
Preview of Latex formulas and images in new ST tooltip

### ISSUE THREAD

**Issue by jlegewie** [LaTeXing/LaTeXing#112]
Hi,

the new development version of ST3 supports tooltip with images (see changelog for recent versions [here](http://www.sublimetext.com/3dev)). `view.show_popup('<img src="res://Packages/Default/Icon.png">')`, for example, shows the sublime text icon in a tooltip. Latexing could format latex formulas on the fly, save the output as an image and preview the formula in a tooltip. Image preview would be even easier insofar as nothing has to be formated. The same might be possible for latex tables but the tooltip might get to small.

In any case, this would be a great feature!

**Comment by rbialon** [comment]
This would require to translate the specific LaTeX part with all used packages and defined commands in context. The formula might look was different than the expected outcome or not translate at all because of missing packages (even local ones in the .tex-file's folder).
I think that this too much overhead for a tooltip. At least I couldn't find a fast and reliable way.

**Comment by csch0** [comment]
Yes I agree with @rbialon. Something similar is done with the tikz support or the partial mode, so we will see but don't place this high on the todo list tbh.
